src/Symfony/Bundle/SecurityBundle/Resources/config/security.xml

```diff
@@ -96,6 +96,7 @@
         <service id="security.role_hierarchy" class="Symfony\Component\Security\Core\Role\RoleHierarchy">
             <argument>%security.role_hierarchy.roles%</argument>
         </service>
+        <service id="Symfony\Component\Security\Core\Role\RoleHierarchy" alias="security.role_hierarchy" />
```

Comment:
Should be `RoleHierarchyInterface` to me

Reply:
Arf, I noticed that too, but it looks like I forgot to push. Thanks.